api/src/test/java/bio/terra/tanagra/service/search/SqlVisitorTest.java

```diff
@@ -113,6 +132,22 @@ void filterArrayFunctionThrowsIfOperandsEmpty() {
         () -> Filter.ArrayFunction.create(ImmutableList.of(), Filter.ArrayFunction.Operator.AND));
   }
 
+  @Test
+  void filterRelationship() {
+    assertEquals(
+        "p.id IN (SELECT a.person_id FROM foo.address AS a WHERE a.zip_code = 12345)",
```

Comment:
demonstrating a case where the outer variable is used within the filter may be good coverage to have as well

Reply:
done.